common/xml/projects.xml

```diff
@@ -0,0 +1,214 @@
+		</profileDesc>
+		<revisionDesc>
+			<change when="2022-10-25" who="#jhf">Created file</change>
+			<change when="2022-10-30" who="#brg">Added list of projects to test xml:id naming
+				convention</change>
+		</revisionDesc>
+	</teiHeader>
+	<text xml:lang="en">
+		<front>
+			<div>
+				<head>About the DHQ Project Registry</head>
+				<p>This is an initial, local registry of digital humanities projects that are
+					referenced from within DHQ articles. DHQ is working in partnership with other
+					publications (in particular, Reviews in Digital Humanities) to develop a broader
+					and more public project registry, possibly using Wikidata or other public linked
+					open data platforms, to which this registry will refer. Currently this registry
+					exists simply to create a target for project references and enable
+					identification and disambiguation.</p>
+			</div>
+			<div>
+				<head>Usage and expansion</head>
+				<p>Each DHQ article contains in its metadata one or more project identifiers which
+					in turn reference items in this registry. These project identifiers will be used
+					in several ways within the DHQ publishing system: <list>
+						<item>in the DHQ search interface, to enable readers to search for articles
+							that reference or discuss a specific project</item>
+						<item>to generate a project index that groups together articles based on
+							project references</item>
+						<item>to support analytics and visualizations</item>
+					</list>
+				</p>
+				<p>When an article is accepted for publication, as part of the encoding process the
+					managing editors will encode an inventory of project references in the article
+					metadata. Any new projects will be added to the registry.</p>
+			</div>
+			<div>
+				<head>Definition of <term>digital humanities project</term></head>
+				<p>Criteria for characterizing something as a digital humanities project: 
+						<list>
+							<item>goal that goes beyond a singular output</item>
+							<item>an organization that continues after the output has been achieved</item>
+							<item>typically (but not necessarily) more than one person</item>
+							<item>an organization, not solely a publication</item>
+						</list>
+				</p>
+			</div>
+		</front>
+		<body>
+			<listOrg>
+				<!-- use @cert with value "low" in cases where we're not sure whether the entity is actually a "project" according to our definition -->
+				<org xml:id="" cert="['low' if not certain]">
+					<name>[project name here]</name>
+					<ptr type="project_site" target=""/>
+				</org>
+			</listOrg>
+
+			<listOrg>
+				<org xml:id="">
+					<name>Lincoln/Net</name>
+					<!-- article 000003 -->
+					<ptr type="project_site" target="http://lincoln.lib.niu.edu"/>
+					<!-- url given in article -->
+					<ptr type="project_site" target="https://digital.lib.niu.edu/illinois/lincoln"/>
+					<!-- current url -->
```

Comment:
How is software supposed to figure out which pointer is for the URL as given in the article vs the “current” URL? And how is either software or a human supposed to know what “current” means?

My knee-jerk reaction is to add the TEI `@versionDate` attribute to the so-called “current” `<ptr>`. I admit it is mild attribute abuse, since [`@versionDate`](https://tei-c.org/Vault/P5/4.5.0/doc/tei-p5-doc/en/html/ref-att.translatable.html) is intended to be used for translations. And some would say the current pointer isn’t really a _version_ of the one given in the article. So maybe creating a new `@as-of` is the way to go. Either way, having the date that a particular pointer worked could prove useful.